src/app/contribute/overview/index.html

```diff
@@ -23,80 +17,85 @@
       </stache-tutorial-step-heading>
       <stache-tutorial-step-body>
         <p>
-          Before you start any work, file an issue. If an issue already exists, comment on the issue and tag the SKY UX team so we know you intend to work on it. The sooner we can discuss potential code changes, the better. To file a issue with SKY UX, follow these instructions:
+          Before you start any work, <a stacheRouterLink="/contribute/overview/file-issue">file an issue</a>. If an issue already exists, comment on it and tag the SKY UX team so that we know you intend to work on it. The sooner we discuss potential code changes, the better.
+        </p>
+        <p>
+          SKY UX provides functionality through multiple repositories that target different areas of development, so use the <a stacheRouterLink="/contribute/packages">packages matrix</a> to find the appropriate repo to file your issue.
         </p>
-
-        <ol>
-          <li>
-            Create an issue in the <a stacheRouterLink="/contribute/packages">applicable repo</a>.
-          </li>
-          <li>
-            Delete the default fields for <a stacheRouterLink="../report-bugs">reporting bugs</a> and describe the contribution and potential code changes to SKY UX.
-          </li>
-          <li>
-            For new components or significant features, you must include design documents and requirements.
-          </li>
-          <li>
-            Subscribe to the issue to receive notifications.
-          </li>
-        </ol>
       </stache-tutorial-step-body>
     </stache-tutorial-step>
-
+  
     <stache-tutorial-step>
-     <stache-tutorial-step-heading>
-        Discuss with the SKY UX team
+      <stache-tutorial-step-heading>
+        Discuss the contribution with the SKY UX team
       </stache-tutorial-step-heading>
       <stache-tutorial-step-body>
-        Once the issue is ready, meet with the SKY UX team to discuss architecture and implementation details, requirements, and timelines. A SKY UX developer will be assigned to the issue; they can help answer any questions you may have.
-      </stache-tutorial-step-body>
+        <p>
+          After you <a stacheRouterLink="/contribute/overview/file-issue">file an issue</a>, discuss the architecture and implementation details, requirements, and timelines with the SKY UX team. We will assign a developer to your issue to help answer questions.
+        </p>
+        <p>
+          Keep in mind that the Blackbaud design council must approve changes that impact the UI, such as new components or changes to existing components. Contributions that do not have design approval must include detailed design specifications. The design approval process can take significant time if components are complex or designs require significant changes, so account for that in your development timeline. If you need help with the design approval process, ask the SKY UX team.
```

Comment:
I like this blurb but I think it might make more sense in the previous step, filing an issue! The design should be part referenced in the issue and I think is often necessary for the implementation/architecture discussion

Reply:
I'll move this up and also tighten step 1 up so that it isn't too long.